code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -117,37 +117,25 @@
 	robotize()
 	..()
 
-/obj/item/organ/internal/optical_sensor
+/obj/item/organ/internal/eyes/optical_sensor
```

Comment:
I'll leave a "change requested" so that this doesn't get glossed over in review.